internal/pkg/manifest/validate_test.go

```diff
@@ -24,13 +24,14 @@ import (
 	"github.com/slackapi/slack-cli/internal/logger"
 	"github.com/slackapi/slack-cli/internal/shared"
 	"github.com/slackapi/slack-cli/internal/shared/types"
+	"github.com/slackapi/slack-cli/internal/slackcontext"
 	"github.com/slackapi/slack-cli/internal/slackerror"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 )
 
 func Test_ManifestValidate_GetManifestLocal_Error(t *testing.T) {
-	ctx, clients, _, log, appMock, authMock := setupCommonMocks()
+	ctx, clients, _, log, appMock, authMock := setupCommonMocks(t)
```

Comment:
note: It's not obvious that this change adds a `t *testing.T` argument to `setupCommonMocks(t)`. Same reason as before, so that we can call `slackcontext.MockContext(t.Context())`.